### ISSUE TITLE
"Connection error. You may have been banned." for a couple days now.

### ISSUE THREAD

**Issue by patq911** [zixaphir/appchan-x#848]
I've been getting this error for a couple days now, on all boards.  I've tried resetting the settings.  The only way I can post is if I disable appchanx and use the default no extensions (not even the included extension with 4chan).  If I use the included extension it says "connection error".  I'm not banned.

I am using Version 41.0.2267.0 dev-m of Chrome, that's usually the problem when I have an issue with an extension. (I have to use Chrome and the dev version for reasons).

I just did a test and Firefox worked perfectly, so I'm fairly sure it's Chrome that's fucked up.

**Comment by ccd0** [comment]
Any difference between HTTP and HTTPS?
Any error messages in the Javascript console?
Does the error persist if all other extensions (not just 4chan ones) are disabled?

But yeah, if the native extension is also broken, it's probably a browser bug.

**Comment by patq911** [comment]
No difference between http and https

error of javascript is this: 

    POST https://sys.4chan.org/vg/post net::ERR_BLOCKED_BY_CLIENTscript.js:2632 (anonymous function)script.js:9455 QR.submit

Well this is strange, I disabled every extension except appchanx, and it worked, then I enabled every extension except adblock, and it still worked. But when I enabled adblock, it worked fine.  So what ever it was seems to have fixed itself by disabling those extensions for some reason. I'll comment back in a day or so if it stopped working again.
